.travis.yml

```diff
@@ -8,13 +8,15 @@ env:
   matrix:
   - BUILDTOOL=autotools
   - BUILDTOOL=cmake
-  - BUILDTOOL=autotools_old_compilers
```

Comment:
It was the opposite. The script should have still been added :) The idea of the target is to get an old gcc compiler and compile it with that one, or even a bunch of old compilers. The problem is in implementation, which is I guess it would need to be done by downloading the compiler source, compile the compiler and then run CppUTest. I'm afraid travis people wouldn't like that (and haven't contacted them about it yet), so I left the target there as a reminder to continue that work.

Now the reminder is deleted :P